R/conclass.R

```diff
@@ -224,11 +269,11 @@ Conos <- R6::R6Class("Conos", lock_objects=FALSE,
         }
 
         
-        if(verbose) cat(".")
-        return(data.frame('mA.lab'=rownames(mnn)[mnn@i+1],'mB.lab'=colnames(mnn)[mnn@j+1],'w'=mnn@x,stringsAsFactors=FALSE))
+        if(verbose) message(".")
+        return(data.frame('mA.lab'=rownames(mnn)[mnn@i+1],'mB.lab'=colnames(mnn)[mnn@j+1],'w'=mnn@x, stringsAsFactors=FALSE))
       },n.cores=self$n.cores,mc.preschedule=TRUE)
 
-      if(verbose) cat(" done\n")
+      if(verbose) message(" done\n")
```

Comment:
Other places, perhaps, could be replaced with messages, but "\n" should be removed

Reply:
Agreed. I've removed `"\n"` from `message()` calls